src/main/java/com/github/mauricioaniche/ck/metric/RFC.java

```diff
@@ -29,22 +25,36 @@ private void count(String methodName, IMethodBinding binding) {
 			methodInvocations.add(methodName);
 		}
 	}
-	
-	public void visit(SuperMethodInvocation node) {
-		IMethodBinding binding = node.resolveMethodBinding();
-		count(node.getName()  + "/" + arguments(node.arguments()), binding);
+
+	private String arguments(List<?> arguments) {
+		if(arguments==null || arguments.isEmpty()) return "0";
+		return "" + arguments.size();
 	}
 
 	private String getMethodName(IMethodBinding binding) {
```

Comment:
Oh, great catch here. I think we don't need this method anymore, as maybe can use the `JDTUtils#getMethodFullName` method!

Reply:
Yes, sure!